scripts/add_export.py

```diff
@@ -129,14 +129,31 @@ def add_export(n, hydrogen_buses_ports, export_profile):
     elif snakemake.params.store == False:
         pass
 
-    # add load
-    n.add(
-        "Load",
-        "H2 export load",
-        bus="H2 export bus",
-        carrier="H2",
-        p_set=export_profile,
-    )
+    if snakemake.params.export_endogenous == False:
```

Comment:
Since `snakemake.params.export_endogenous` we should be able to use it directly as a condition, without `== False`